aperturedb/Entities.py

```diff
@@ -153,10 +203,31 @@ def get_connected_entities(self,  type: EntityType, constraints: Constraints = N
                 }
             ]
             res, r, b = execute_batch(query, [], self.db)
-            result.append(self.known_entities[type.value](
-                db=self.db, response=r[1]["FindEntity"]["entities"], type=type.value))
+            cl = Entities
+            if entity_class in self.known_entities:
+                cl = self.known_entities[entity_class]
+            result.append(cl(
+                db=self.db, response=r[1]["FindEntity"]["entities"], type=entity_class))
         return result
 
+    def get_blob(self, entity) -> Any:
+        query = [
+            {
+                "FindVideo": {
```

Comment:
why "FindVideo"?

Reply:
This should be dynamic, I will address it.